src/transformers/training_args.py

```diff
@@ -571,17 +557,20 @@ class TrainingArguments:
             Whether or not to compile the model using PyTorch 2.0
             [`torch.compile`](https://pytorch.org/get-started/pytorch-2.0/) (requires a nighlty install of PyTorch).
 
-            If set, the backend will default to `"inductor"` (can be customized with `torch_compile_backend`) and the
-            mode will default to `"default"` (can be customized with `torch_compile_mode`).
+            This will use the best defaults for the [`torch.compile`
+            API](https://pytorch.org/docs/2.0/generated/torch.compile.html?highlight=torch+compile#torch.compile). You
```

Comment:
should we add an Issue to update this doc to use:
https://pytorch.org/docs/stable/generated/torch.compile.html?highlight=torch+compile#torch.compile
once pt-2.0 is released on Wed?

since we don't want to link to a hardcoded version of the doc.

but it's 404 at the moment.

Reply:
We can do it in a couple of weeks, yes.

Reply:
added https://github.com/huggingface/transformers/issues/22142 so not to forget